test.py

```
@@ -1,5 +1,40 @@
+            d.append(i * a)
+        else:
+            d.append(i * b)
+    for i in range(10):
+        if i % 2 == 0:
+            print(i * a)
+        else:
+            print(i * b)
+    return d
+
+def calc_area(r):
+    return 3.14 * r * h
```

Ruff (0.12.2)

Undefined name `h`

(F821)